CWE/events/tanganyika.txt

```diff
@@ -68,116 +70,190 @@ country_event = {
 
 }
 
-#Tanganyika Autonomy
+#Tanganyika Independence Act 1961
```

Comment:
This event will be restored to its original as TAN does not start as a puppet.

Reply:
I'd rather keep this but tweak the conditions and effects to make it released and independent instead, UN had ruled that by 1961/2 it should have been independent so no reason to release it as a puppet five years before, just gonna keep XEF for puppet route

Reply:
I feel that it is better to keep the current approach as it is more realistic. Direct rule ended in 1958 when real elections were held.